docs/source/models.rst

```diff
@@ -139,48 +139,134 @@ For more information, see the :mod:`mlflow.pyfunc` documentation.
 H\ :sub:`2`\ O (``h2o``)
 ^^^^^^^^^^^^^^^^^^^^^^^^
 
-The H2O model flavor enables logging and loading H2O models. These models will be saved by using the :py:mod:`mlflow.h2o.save_model`. Using :py:mod:`mlflow.h2o.log_model` will also enable a valid ``Python Function`` flavor.
+The ``h2o`` model flavor enables logging and loading H2O models. 
 
-When loading a H2O model as a PyFunc model, :py:mod:`h2o.init(...)` will be called. Therefore, the right version of h2o(-py) has to be in the environment. The arguments given to :py:mod:`h2o.init(...)` can be customized in ``model.h2o/h2o.yaml`` under the key ``init``. For more information, see :py:mod:`mlflow.h2o`.
+The :py:mod:`mlflow.h2o` module defines :py:func:`save_model() <mlflow.h2o.save_model>` and 
+:py:func:`log_model() <mlflow.h2o.log_model>` methods for saving H2O models in MLflow model format.
+These methods produce MLflow models with the ``python_function`` flavor, allowing them to be loaded 
+as generic Python functions for inference via :py:func:`mlflow.pyfunc.load_pyfunc()`. When MLflow 
+models with the ``h2o`` flavor are loaded via :py:func:`load_pyfunc() <mlflow.pyfunc.load_pyfunc>`, 
```

Comment:
When MLflow  models with the ``h2o`` flavor are loaded via -> When you load MLflow models with the ``h2o`` flavor are loaded using

Reply:
Fixed!